javaslang/src/main/java/javaslang/collection/TreeMultimap.java

```diff
@@ -194,7 +205,7 @@ private Builder(ContainerType containerType, SerializableSupplier<Traversable<?>
 
     /**
      * Narrows a widened {@code HashMultimap<? extends K, ? extends V>} to {@code HashMultimap<K, V>}
-     * by performing a type safe-cast. This is eligible because immutable/read-only
+     * by performing a type-safe cast. This is eligible because immutable/read-only
```

Comment:
I'm not very used to rebasing. Did try it but the Atom editor had a problem and the interactive mode didn't work. Found a fix but using now a guy. There I squash/merge/cherry-pick and delete all commits manually.

From what I can see now, this will not disappear because I did it in a commit on this branch.